services/web/src/stores/session.js

```diff
@@ -106,6 +113,34 @@ export class SessionProvider extends React.PureComponent {
     this.setStored({});
   };
 
+  // Organizations
+
+  loadOrganization = async () => {
+    const organizationId = this.getStored('organizationId');
+    if (organizationId) {
+      try {
+        const { data } = await request({
+          method: 'GET',
+          path: `/1/organizations/${organizationId}`,
+        });
+        return data;
+      } catch(err) {
+        this.removeStored('organizationId');
```

Comment:
i assume it would only be removed if 401.

Reply:
Why is that? I think technically anything not a 5xx should be removed no?

Reply:
well perhaps but generally speaking 500 is an api problem, and we should set to fix any 500, but perhaps fine in this case ... hard to tell if thats right move to avoid stuck users.

But right now even if the client internet connection fails then we remove the organization, 
if we deploy the api at bad moment and we return 502 

I can specially see this is gonna be annoying to work with locally, where we restart the api alot.

Reply:
Well this is the client right? For the client's purposes a 5xx level problem means that the issue might be resolved, anything else means it's a client (or resource) issue and would likely happen again... in this case it's on app bootstrap which could send the app into an infinite reload loop which is specifically what I'm trying to avoid here so I think anything 4xx level should remove.

Reply:
updated to only remove on 4xx